tests/test_qdrant_client.py

```diff
@@ -79,7 +80,7 @@ def test_qdrant_client_integration(prefer_grpc):
 
     # Create payload index for field `random_num`
     # If indexed field appear in filtering condition - search operation could be performed faster
-    index_create_result = client.create_payload_index(COLLECTION_NAME, "random_num")
+    index_create_result = client.create_payload_index(COLLECTION_NAME, "random_num", PayloadSchemaType.FLOAT)
```

Comment:
I understand that the name `PayloadSchemaType` is a convention from the previous API with restricted data types, but it might be a good idea to rename it to `PayloadIndexType` in a future release before v1.0.

Reply:
There should be no problem to rename, will look at it

Reply:
`PayloadIndexType` is already taken by the type of index itself :disappointed:, might be `PayloadSchemaType` is not too bad? :thinking:

Reply:
`PayloadSchemaType` is not too bad

I think so. Let's release v0.7 as is, and we may think of a workaround in a future release if we observe a confusion among users?